### PR TITLE
Don't assume m is non-nil

### PR DESCRIPTION
**Pull request by siggisim**
**Related issues**: N/A

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]
thanks

File: server/util/filter/filter.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func invocationMetricToDbField(m stat_filter.InvocationMetricType, paramPrefix s
 }
 
 func MetricToDbField(m *stat_filter.Metric, paramPrefix string) (string, error) {
-	if m.Invocation != nil {
+	if m.GetInvocation() != nil {
 		return invocationMetricToDbField(m.GetInvocation(), paramPrefix)
 	} else if m.Execution != nil {
```
Comment (jdhollen) [inline server/util/filter/filter.go:59]: should do this one too

Comment (vadimberezniker) [inline server/util/filter/filter.go:59]: what about m.Execution

**Review by jdhollen — APPROVED** [review]

**Review by tylerwilliams — APPROVED** [review]

File: server/util/filter/filter.go (hunk 1 above)
Comment (siggisim) [reply server/util/filter/filter.go:62]: I did, but then go yelled at me so I took another approach that should fix both.
```
server/util/filter/filter.go:57:26: invalid operation: m.GetInvocation() != nil (mismatched types stat_filter.InvocationMetricType and untyped nil)
server/util/filter/filter.go:59:32: invalid operation: m.GetExecution() != nil (mismatched types stat_filter.ExecutionMetricType and untyped nil)
```

**Pull request merged**